src/installer/managed/Microsoft.NET.HostModel/AppHost/HostWriter.cs

```diff
@@ -233,6 +239,32 @@ void FindBundleHeader()
             return headerOffset != 0;
         }
 
+        private static void CodeSign(string args)
+        {
+            if (!RuntimeInformation.IsOSPlatform(OSPlatform.OSX))
+                return;
+
+            const string codesign = @"/usr/bin/codesign";
+            if (!File.Exists(codesign))
+                return;
+
+            var p = new Process()
+            {
+                StartInfo = new ProcessStartInfo()
+                {
+                    Arguments = args,
```

Comment:
You do not need ProcessStartInfo and any of these arguments for a simple process start. You can just use the simple `Process.Start(string fileName, string arguments)` method.